src/resolve/resolved-bus.c

```diff
@@ -467,7 +467,7 @@ static int bus_method_resolve_hostname(sd_bus_message *message, void *userdata,
         _cleanup_(dns_question_unrefp) DnsQuestion *question_idna = NULL, *question_utf8 = NULL;
         _cleanup_(dns_query_freep) DnsQuery *q = NULL;
         Manager *m = userdata;
-        const char *hostname;
+        char *hostname;
```

Comment:
Why `const` is dropped?

Reply:
It should not have been dropped, I've added it back now